renovate.json

```diff
@@ -0,0 +1,13 @@
+{
+    "extends": [
+        "github>konflux-ci/mintmaker//config/renovate/renovate.json"
+    ],
+    "dockerfile": {
+        "fileMatch": ["^Dockerfile\.[a-zA-Z0-9\.-]+$"]
+    },
+    "ignorePaths": ["drift-cache"],
+    "ignoreDeps": [
+        "registry.redhat.io/openshift4/ose-operator-registry",
+        "registry.redhat.io/openshift4/ose-operator-registry-rhel9"
```

Comment:
Why do ignore these dependencies?

Reply:
Gotten from a sample of an openshift app, but again we can keep for now the standards.